<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HMS-LP-N-47" failures="9" tests="10" name="Test" time="38.158" errors="0" timestamp="2025-02-09T23:44:25 IST">
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="1.368">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.verify_loginDDT(TC_003_LoginDataDrivenTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="12.897">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testCases.TC_003_LoginDataDrivenTest.verify_loginDDT(TC_003_LoginDataDrivenTest.java:81)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="11.122"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="0.416">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.verify_loginDDT(TC_003_LoginDataDrivenTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="0.024">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.verify_loginDDT(TC_003_LoginDataDrivenTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="0.025">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.verify_loginDDT(TC_003_LoginDataDrivenTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="0.014">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.verify_loginDDT(TC_003_LoginDataDrivenTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="0.012">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.verify_loginDDT(TC_003_LoginDataDrivenTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="0.02">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.verify_loginDDT(TC_003_LoginDataDrivenTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="verify_loginDDT" time="0.061">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_003_LoginDataDrivenTest.verify_loginDDT(TC_003_LoginDataDrivenTest.java:88)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
